src/main/resources/hudson/plugins/performance/PerformancePublisher/config.jelly

```diff
@@ -14,208 +14,208 @@
   </f:entry> 
 
   <f:entry>
-    <table style="border:1;border-style:solid"> 
+    <div style="border:1;border-style:solid">
```

Comment:
@jsoref 
I think this change produce chaos in UI
https://prnt.sc/mr881e

Reply:
Current UI  https://prnt.sc/mr89e0

Reply:
Oops, that's for later, dropped for now